README.md

```diff
@@ -191,5 +222,5 @@ Next checks whether the image url which loader generates has the exact value whi
 [test-status-img]: https://github.com/uploadcare/nextjs-loader/actions/workflows/test-and-lint.yml/badge.svg
 [npm-img]: https://img.shields.io/npm/v/@uploadcare/nextjs-loader.svg
 [npm-link]: https://www.npmjs.com/package/@uploadcare/nextjs-loader
-[demo-link]: https://uploadcare-nextjs-loader.vercel.app/
+[demo-link]: https://uploadcare-nextjs-loader.netlify.app/
```

Comment:
We can't deploy to Vercel on CI because their free plan doesn't work with organizations. Before that, I deployed to Vercel using my personal account.

Reply:
Got it. Thanks